tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+FROM_TAG="$1"
```

Comment:
Can we please detect the from tag automatically? It really doesn't matter what the current tag is, we just want to update whatever is there.
```suggestion
FROM_TAG=$(grep "git clone --single-branch" Documentation/Getting-Started/quickstart.md | awk '{ print $6 }')
```

Reply:
I'm not sure this is so simple when we cut a `v1.Y.0-beta.0` tag from `master` when it's already been branched to `release-v1.Y`. It seems like it might be good to add autodetection later and focus first on getting the find/replace logic correct.

Reply:
We don't need to look at the branch name, I think it really is simple if we just extract the value in the 6th position from that grep command.

Reply:
To be clear, we aren't running a `git` command, we are just grepping for a line with "git clone".